logging.go

```diff
@@ -32,9 +33,9 @@ type LogOptionsSetter interface {
 
 // LogToFileWith does allows to call LogToFile with a custom LogOptionsSetter.
 func LogToFileWith(path string, prefix string, log LogOptionsSetter) (*os.File, error) {
-	f, err := os.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0o644)
+	f, err := os.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0o644) //nolint:gomnd
```

Comment:
Actually, this might be a valid warning and `0o600` could be more sensible for (potentially) sensitive data like a logfile?

Reply:
Good call, I think that makes sense for this use case.

Regardless, this is still a magic number warning. I had originally pulled the number into a const but found trying to name a token describing `0o644` less clear than the literal representation. So, regardless, I’d probably keep the `nomnd` directive.